Framework/Algorithms/test/MergeRunsTest.h

```diff
@@ -1102,46 +1102,46 @@ class MergeRunsTest : public CxxTest::TestSuite {
   }
 
   void test_mergeSampleLogs_fail_where_params_are_different_fails() {
-    std::string mergeType = SampleLogsBehaviour::FAIL_MERGE;
+    const std::string mergeType = MergeRuns::FAIL_MERGE;
     auto ws = create_group_workspace_with_sample_logs<double>(
         mergeType, "prop1", 1.0, 2.0, 0.0, 0.0);
     do_test_mergeSampleLogs(ws, "prop1", mergeType, "1", 1);
   }
 
   void
   test_mergeSampleLogs_fail_where_params_are_different_but_inside_tolerance_succeeds() {
-    std::string mergeType = SampleLogsBehaviour::FAIL_MERGE;
+    std::string mergeType = MergeRuns::FAIL_MERGE;
```

Comment:
Can this `std::string` and below also be made `const`?

Reply:
Good idea.